pkgs/data/fonts/input-fonts/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "input-fonts-${version}";
+  version = "2016-04-14"; # date of most recent file in downloaded archive
+
+  src = fetchurl {
+    name = "${name}.zip";
+    url = "http://input.fontbureau.com/build/?fontSelection=whole&a=0&g=0&i=0&l=0&zero=0&asterisk=0&braces=0&preset=default&line-height=1.2&accept=I+do&email=";
+    sha256 = "01hs12j74hfys2ki6gv4hig54xhgi0vww0i2imsarnxkaksf8ccl";
```

Comment:
This was more a question for the other people, but the problem is, that the file could change which would break the hash.